cms/templates/base.html

```diff
@@ -84,7 +84,7 @@
                       url_name: "${context_course.location.name | h}",
                       org: "${context_course.location.org | h}",
                       num: "${context_course.location.course | h}",
-                      display_course_number: "${_(context_course.display_coursenumber)}",
+                      display_course_number: "${_(context_course.display_number_with_default)}",
```

Comment:
`context_course.display_coursenumber` is None here due to which encoding fails

@doctoryes Any concerns?